modules/ob-creating-a-s2i-build.adoc

```diff
@@ -44,16 +47,37 @@ EOF
 <2> The build strategy that you use to build the container.
 <3> A name-value list to specify values for parameters defined in the build strategy. To set the value of the `builder-image` strategy parameter, specify the builder image location required to build the output image.
 <4> The location where the built image is pushed. You can push the built image to a custom Quay repo. `pushSecret` is the secret name that stores the credentials for pushing container images. To generate a secret of the type `docker-registry` for authentication, see _Authentication to container registries_ in the _Additional resources_ section.
++
+.Using `shp` CLI
+[source,terminal]
+----
+$ shp build create s2i-nodejs-build \
+--strategy-name="source-to-image" \
+--source-url="https://github.com/shipwright-io/sample-nodejs" --source-context-dir="source-build" \
+--builder-image="quay.io/centos7/nodejs-12-centos7" \
+--output-image="quay.io/<repo>/s2i-nodejs-example" \ <1>
+--output-credentials-secret="registry-credential" <2>
+----
+<1> Replace `repo` with a valid Quay organization or your Quay user name.
+<2> `output-credentials-secret` is the secret name that stores the credentials for pushing container images. To generate a secret of the type `docker-registry` for authentication, see _Authentication to container registries_ in the _Additional resources_ section.
```

Comment:
An additonal res. section could be plonked directly after the Creating a source-to-image build module. On the Customer Portal, multi-page view might make identifying the add resources difficult as it is currently located at the end of the assembly.